src/System.Management.Automation/engine/runtime/Binding/Binders.cs

```diff
@@ -2029,36 +2029,38 @@ private static object NullRule(CallSite site, object obj)
 
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-windows`
TabCompletion.Miscellaneous completion tests.Test history completion
```powershell
Expected strings to be the same, but they were different.
Expected length: 23
Actual length:   633
Strings differ at index 0.
Expected: 'Test history completion'
But was:  '$env:POWER...'
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\TabCompletion\TabCompletion.Tests.ps1: line 909
909:             $res.CompletionMatches[0].CompletionText | Should -BeExactly "Test history completion"
```